webserver/templates/macros.html

```diff
@@ -13,38 +13,37 @@
 {% endmacro %}
 
 {% macro pager(links, prevpage, current_page, last_page, nextpage) %}
```

Comment:
`prevpage` here could be renamed to `prev_page` to be consistent with other arguments.